integration_tests/integration/approveLicence.cy.ts

```diff
@@ -81,6 +84,80 @@ context('Approve a licence', () => {
     approvalCasesPage2.signOut().click()
   })
 
+  it('should redirect to approve case page on return to list click', () => {
+    cy.task('stubGetPrisonUserCaseloads', singleCaseload)
+    cy.signIn()
+    const indexPage = Page.verifyOnPage(IndexPage)
+    const approvalCasesPage = indexPage.clickApproveALicence()
+    const approvalViewPage = approvalCasesPage.clickApproveLicence()
+    const approvalCasesPage2 = approvalViewPage.clickReturnToList()
+    approvalCasesPage2.getTableRows().should(rows => {
```

Comment:
I think you can verify it's gone to the correct page by checking the url using something like

cy.url().should('eq', 'http://localhost:3007/licence/approve/cases')